resources/views/partials/content-page-stories.blade.php

```diff
@@ -0,0 +1,30 @@
+<article class="card pcc-story-card">
+  <figure>
+    @if(has_post_thumbnail())
+      {!! get_the_post_thumbnail(get_the_ID(), 'small') !!}
+    @else
+    <div class="placeholder-wrap">
+      @svg('mark', ['class' => 'placeholder'])
+    </div>
+    @endif
+  </figure>
+  <div class="pcc-story__details">
+    <header class="text">
+      <h2 class="title">
+        <a href="{{ get_permalink() }}">{{get_post_meta (get_the_id(), 'pcc_story_organization', true)}}</a>
+      </h2>
+    </header>
+    <div class="pcc-story__meta">
+
+      <div class="card__region">
+        <span class="screen-reader-text">Region: </span>
+        <svg class="icon icon--location" aria-hidden="true" viewBox="0 0 20 20" focusable="false">
+            <use href="{{ App\asset_path('images/location.svg#location') }}" />
+        </svg>
+        @foreach (Page::storyRegions(get_the_ID()) as $region)
```

Comment:
This controller method is fetching data about an individual story, not a page. I'd recommend following the pattern shown here:

1. Create a `trait` for stories: https://github.com/platform-coop-toolkit/coop-library/blob/dev/app/Controllers/Partials/Resource.php
2. In the `Page` and `SinglePccStory` controllers, import the trait with a `use` statement:
    - https://github.com/platform-coop-toolkit/coop-library/blob/dev/app/Controllers/Single.php
3. In the page template, call the methods using `Page::methodName()` and `SinglePccStory::methodName()` respectively:
    - https://github.com/platform-coop-toolkit/coop-library/blob/dev/resources/views/partials/content-lc_resource.blade.php
    - https://github.com/platform-coop-toolkit/coop-library/blob/dev/resources/views/partials/content-single-lc_resource.blade.php

We can go over this if you like. The advantage is you don't need to pass the ID and you can reuse the same method in both contexts.